ci/verify_env.jl

```diff
@@ -0,0 +1,40 @@
+"""
+The script checks, if custom dependency for unit tests via `CI_UNIT_PKG_URL_<package_name>` are set.
+If yes, the script fails, which means in practice, that the PR rely on non merged code.
+"""
+
+"""
+    extract_env_vars_from_git_message!()
+
+Parse the commit message, if set via variable `CI_COMMIT_MESSAGE` and set custom urls.
+"""
+function extract_env_vars_from_git_message!()
+    if haskey(ENV, "CI_COMMIT_MESSAGE")
+        println("Found env variable CI_COMMIT_MESSAGE")
```

Comment:
Consider using `@info` instead of `println`, because this works better with the logging module (cf. [here](https://docs.julialang.org/en/v1/stdlib/Logging/)).